english/check-your-phone/check-your-phone.md

```diff
@@ -6,7 +6,7 @@ But you dont know whether to trust yourself.
 As you look at your phone, you realize that you can't make
 out what it says....
 
-You look up and suddenly, there is a woman in front of you 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.